notifications.bs

```diff
@@ -601,6 +604,39 @@ or vibrating the device again, unless the <a>renotify preference flag</a> is set
 </ol>
 
 
+<h3 id=selecting-an-icon-url>Selecting an icon URL</h3>
+
+<p>The <dfn>icon URL selection steps</dfn> for selecting a URL given a list <var>icons</var> of type
+{{NotificationIcon}}, a <var>fallbackURLString</var> of type {{USVString}}, and
+<a lt="base url">base URL</a> <var>baseURL</var> are:
+
+<ol>
+  <li><p>Let <var>selectedURL</var> and <var>selectedIcon</var> be null
+
+  <li><p>For each <var>icon</var> in <var>icons</var> run these substeps:
+
+  <ol>
```

Comment:
I know we have this indentation elsewhere, but if the `<p>` is not the sole child of an `<li>` it really should go on its own line and get indented. And this `<ol>` too.